openlibrary/templates/account/activity_feed.html

```diff
@@ -0,0 +1,62 @@
+$def with(feed, feed_url, follows_others)
+
+$# Renders the given feed in a container filled with cards.
+$#
+$# * feed (list)              -- List of recent bookshelf events
+$# * feed_url (string)        -- `href` for the feed's source
+$# * follows_others (boolean) -- `True` if the authenticated patron follows at least one person
+
+$code:
+    shelf_id_to_name = {
+        1: _('Want to Read'),
+        2: _('Currently Reading'),
+        3: _('Already Read')
+    }
+
+$def activity_card_footer(shelf_name, occurred_on, username):
+    $ user_key = "/people/%s" % username
+    $ avatar_url = "%s/avatar" % user_key
+    <div class="activity-feed-card__footer">
+        <div class="activity-feed-card__user">
+            <a href="$user_key">
+                <img src="$avatar_url" alt="$_('Avatar of the owner of the list')">
+            </a>
+            <div class="activity-feed-card__footer-copy">
+                $:_('<a class="activity-feed-card__user-link" href="$user_key">@%(username)s</a> added to %(shelf_name)s %(occurred_on)s', username=username, shelf_name=shelf_name, occurred_on=occurred_on)
+            </div>
+        </div>
+        <div class="activity-feed-card__follow-button">
+            $ is_subscribed = ctx.user and ctx.user.is_subscribed_user(username)
+            $ settings = ctx.user.get_users_settings()
+        </div>
+    </div>
+
+$def trending_card_footer(shelf_name, occurred_on, username):
+    <div class="trending-feed-card__footer">
+        $_("%(someone)s marked as %(shelf_name)s %(occurred_on)s", someone=username, shelf_name=shelf_name, occurred_on=occurred_on)
+    </div>
+
+<div class="carousel-section">
+    <div class="carousel-section-header">
+        <h2 class="home-h2">
+            <a href="$feed_url">$_("My Feed")</a>
+        </h2>
+        $if not follows_others:
+            <p>$_("Here's the latest activity around the library. Follow readers to personalize your feed.")</p>
+        $elif not feed:
+           <p>$_("None of the people that you follow have logged books.  When they do, you'll see it here.")</p>
+        <div class="list-showcase">
```

Comment:
```suggestion
        <div class="activity-feed">
```
`list-showcase` is an exceptionally poor name for this activity feed container.